homeassistant/components/cloud/tts.py

```diff
@@ -178,7 +177,18 @@ async def async_get_tts_audio(
         gender: Gender | str | None = options.get(ATTR_GENDER)
         gender = handle_deprecated_gender(self.hass, gender)
         original_voice: str | None = options.get(ATTR_VOICE)
+        if original_voice is None:
```

Comment:
The default voice is only valid for the default language:
```suggestion
        if original_voice is None and language == self._language:
```